spec/services/catalog/add_to_order_spec.rb

```diff
@@ -36,4 +45,11 @@
       expect { described_class.new(params).process }.to raise_error(ActiveRecord::RecordNotFound)
     end
   end
+
+  context "when passing in a x-rh-identity header" do
```

Comment:
@lindgrenj6 
Should we add a text case to do the round trip
Save it as context and fetch it back and run a with_request on it to see that the read value can be decoded. SO we would have to check that the userid or email coming back was ok

https://github.com/ManageIQ/manageiq-api-common/blob/1bf945a647f86df3d9bd9552e288942ceff7e9db/spec/support/user_header_spec_helper.rb#L8

Reply:
Great idea, got a test added for this!